doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,175 @@
+.. _ug_nrf54h20_flpr:
+
+Working with the FLPR core
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *fast lightweight peripheral processor* (FLPR).
+The FLPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API:
+
+* ##List to be completed
```

Comment:
uart is for sure supported (UARTE120)

Reply:
@hubertmis @shanthanordic Any additional inputs?